components/integrations/bots/bots.jsx

```diff
@@ -181,7 +183,7 @@ export default class Bots extends React.PureComponent {
                         defaultMessage='Bot Accounts'
                     />
                 }
-                addText={
+                addText={ this.props.createBots &&
```

Comment:
I assume you tested this and it actually goes away if you pass false here. But it is strange.

Reply:
Yes it has the desired behavior. I wasn't exactly sure how to conditionally render based on the flag.